### PR TITLE
[#48] [Chore] Display taps and gestures on the screen

### LINKED ISSUE #48: [Chore] Display taps and gestures on the screen
## Why

Showing taps and gestures on the screen (either on the simulator or the device) gives the audience a much clearer context of what's happening.
It's very helpful for demos, presentations, or videos.

For this purpose, we can use the [ShowTime](https://github.com/KaneCheshire/ShowTime) library. This is not required, any good alternatives are free to propose and integrate.
Please note that this is enabled on the _debug_ build only.

## Who Benefits?

Developers who:
- present the app demo
- make POW video for their PRs

### PR DESCRIPTION
**Pull request by Thieurom**
- Close #48

## What happened 👀

- Use the [`ShowTime`](https://github.com/KaneCheshire/ShowTime) library
- Config the visual touch with the size of 44pt, with a shade of gray color to display best on both light and dark modes.
- Enable for `DEBUG` only

## Insight 📝

I think it's nice 😄 

## Proof Of Work 📹

| Dark mode | Light mode |
|---|---|
|<video src="https://user-images.githubusercontent.com/17972674/208622679-62a6126d-60db-40f0-b6c9-79f3efd9f18a.mp4">|<video src="https://user-images.githubusercontent.com/17972674/208622751-40fa64a9-324d-4562-b121-940064dfee84.mp4">|

### CODE REVIEW

**Review by nkhanh44 — APPROVED** [review]
LGTM 👍

File: CryptoPrices/Sources/App/CryptoPricesApp.swift [hunk 1]
```diff
@@ -48,5 +49,13 @@ struct CryptoPricesApp: App {
 
     init() {
         Fonts.registerAllCustomFonts()
+
+        // config showing taps and gestures on screen
+        #if DEBUG
+        ShowTime.enabled = .debugOnly
+        ShowTime.fillColor = .lightGray.withAlphaComponent(0.7)
+        ShowTime.strokeColor = .gray
+        ShowTime.disappearDelay = 0.1
```
Comment (minhnimble) [inline CryptoPrices/Sources/App/CryptoPricesApp.swift:58]: Optional: Just a minor improvement suggestion, maybe we can please this setup code for `ShowTime` in a extension and call it in one line here at the `init` like what we did with `Fonts` 🙏

**Review by minhnimble — APPROVED** [review]
Just a small suggestion, the rest LGTM, really nice work 🚀

File: CryptoPrices/Sources/App/CryptoPricesApp.swift (hunk 1 above)
Comment (Thieurom) [reply CryptoPrices/Sources/App/CryptoPricesApp.swift]: Updated in [9016ea9](https://github.com/nimblehq/nimble-crypto-ios/pull/49/commits/9016ea9f52212286dfea6d9037eee9543cba3be1) 👍

**Pull request merged**